### ISSUE TITLE
Remove Italian language from installer

### ISSUE THREAD

**Issue by bovirus** [xvitaly/mhed#232]
### Problem details

@xvitaly 

It seems that in last version of .iss file you removed the Italian language from the instalelr

The new .iss file lost most of improvements like

Adding Italian language
Multilanguage detection option
AppVername not include Version etc.

What's the matter?

### Steps to reproduce

Check installer iss file

### Expected behavior

Restore original Italian language options and improvements.

### Actual behavior

Please check current installer

### Project version

1.5.2

### Platform

Microsoft Windows

### OS version

22 h2 64bit Italian

### Installation type

EXE installer

**Comment by xvitaly** [comment]
Can you elaborate? I didn't delete anything. Only RTF files have been converted to plain text files for better maintainability and added 64-bit ARM support by using new `x64compatible` option.

**Comment by bovirus** [comment]
@xvitaly 

If you are agree I will make a PR with Installer improvements about .iss file.

**Comment by xvitaly** [comment]
Current changes are not final. We're working both on the installer and the application.

You don't need a separate PR. Just rebase it against the latest commit. Once 2.0-rc is ready, we'll try to integrate the changes from your PR.

**Comment by bovirus** [comment]
@xvitaly 

If you are working on the branch and don't integrate my PR changes every time that you made some changes to the structure I have to rebuild the changes.

My prfered solution is this.
You can integrate my PR change in current stable branch.
Then any your future change always contains my PR changes.
